### ISSUE TITLE
Update: jqueryui to v1.13.0

### ISSUE THREAD

**Issue by MagnonD** [cdnjs/packages#889]
### Library name

jqueryui

### Library JSON file

https://github.com/cdnjs/packages/blob/master/packages/j/jqueryui.json

### Library source

https://github.com/jquery/jquery-ui

### Files to include

- jquery-ui.min.css
- jquery-ui.min.js
- jquery-ui.css
- jquery-ui.js

**Comment by MattIPv4** [comment]
👋 It looks like both the jquery-ui GitHub repo & NPM package do not distribute the files you're referencing here, it appears that they only distribute the constituent parts.

**Comment by MagnonD** [comment]
Thanks for looking into it and creating that issue for jquery-ui

**Comment by MattIPv4** [comment]
Going to close this out, the jQuery folks seem uninterested in publishing a version that is compatible with cdnjs.

**Comment by gtbu** [comment]
I already send a mail to the cloudflare service to update jquery-ui to 1.13.0  and they promised.

There are [changes](https://jqueryui.com/changelog/1.13.0/)  which make it incompatible to Elfinder in Typesetter-CMS (before V 1.12.1) : Invalid jQuery UI configuration.
Selectable, draggable and droppable components must be included.

**Comment by MagnonD** [comment]
I believe they have now publishing a version that is compatible with cdnjs in 1.13.1
See https://github.com/jquery/codeorigin.jquery.com/tree/main/cdn/ui/1.13.1
(Or for the issue see https://github.com/jquery/jquery-ui/issues/2011)

**Comment by MattIPv4** [comment]
It looks like they've merged the PR that should generate a version that can be used on cdnjs, but a new library version has not yet been published using that new release mechanic.

The repo you link to (codeorigin) doesn't work for cdnjs as the directory changes each release, and the versions aren't tagged :)

**Comment by MagnonD** [comment]
I am new to how these all link, or on the requirements for, but I thought the 1.13.1 was the new tag.
Is the problem that the tag changing changes the directory?
So https://github.com/jquery/codeorigin.jquery.com/tree/main/cdn/ui/1.13.1 would change to .../1.13.x in future minor releases?

**Comment by MattIPv4** [comment]
Alas no, 1.13.1 was released before their fix for the NPM package was merged, so I imagine it'll work for 1.13.2.

cdnjs needs either a GitHub repository with tags for each version, or an NPM package that is released for each version. In either case, we also expect the assets to be in a consistent location.

The repo you linked doesn't have tags for each version, and it also has a changing directory where the assets are located. The fix that has been merged by the jquery-ui team though should result in the contents of each version within that repository essentially being included as part of each version on NPM, so that we have versioned assets at a consistent location (/dist within the NPM package).
